app/views/invitations/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Invitations#create</h1>
+<p>Find me in app/views/invitations/create.html.erb</p>
```

Comment:
- [ ] You don't need a create view for invitations.
Create is a 'post' action, views are only used for 'get' actions.
Not sure what I mean with 'get' and 'post'? Read[ this article about HTTP methods](https://www.restapitutorial.com/lessons/httpmethods.html).

This file can be safely deleted.